src/ioctls/vcpu.rs

```diff
@@ -93,6 +93,10 @@ pub enum VcpuExit<'a> {
     IoapicEoi(u8 /* vector */),
     /// Corresponds to KVM_EXIT_HYPERV.
     Hyperv,
+    /// Corresponds to an exit reason that is unknown from the current version
+    /// of the kvm-ioctls crate. Let the consumer decide about what to do with
+    /// it.
+    Future(u32),
```

Comment:
Can we have it as `Unknown` instead?

Reply:
I wanted to use `Unknown`, but there's an actual KVM exit reason called `KVM_EXIT_UNKNOWN` :(

Do you have any other idea?

Reply:
Right, I forgot about that one. We can name it `Unsupported`? Returning this as `Unsupported` is not going to bring much information though, it is just a stepping stone I assume until we add the actual variants defined here?

Reply:
Really funny, a known unknown and an unknown unknown:)

Reply:
> Really funny, a known unknown and an unknown unknown:)

Exactly :laughing:

Reply:
>it is just a stepping stone I assume until we add the actual variants defined here?

Yes exactly, but we have to wait for these things to land in upstream Linux kernel before we can update the `kvm-bindings` and `kvm-ioctls`.